addons/block_code/ui/blocks/utilities/background/background.gd

```diff
@@ -46,74 +46,58 @@ func _set_shift_bottom(new_shift_bottom):
 	queue_redraw()
 
 
-func float_array_to_Vector2Array(coords: Array) -> PackedVector2Array:
-	# Convert the array of floats into a PackedVector2Array.
-	var array: PackedVector2Array = []
-	for coord in coords:
-		array.append(Vector2(coord[0], coord[1]))
-	return array
-
-
 func _draw():
 	outline_color = color.darkened(0.2)
 
-	var fill_polygon = [[0.0, 0.0]]
+	var fill_polygon: PackedVector2Array
```

Comment:
I'd think you'd need to initialize this with a constructor like `PackedVector2Array()` or `[]`.

Reply:
One might think that but actually you don't need to.

Reply:
I guess typed variables are automatically initialized to a default value.

https://docs.godotengine.org/en/stable/tutorials/scripting/gdscript/gdscript_basics.html#initialization-order